cmd/restic/cmd_backup.go

```diff
@@ -553,7 +561,7 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 	}
 	t.Go(func() error { return sc.Scan(t.Context(gopts.ctx), targets) })
 
-	arch := archiver.New(repo, targetFS, archiver.Options{})
+	arch := archiver.New(repo, targetFS, archiver.Options{FileReadConcurrency: backupOptions.FileReadConcurrency})
```

Comment:
here you should add a check if `backupOptions.FileReadConcurrency == 0` and if, set to `filereadconcurrency`

Reply:
This is supposed to be handled by the clobbered code.